README.md

```diff
@@ -30,7 +30,7 @@ Web application.
 * Good flexibility and extensibility of materials with node-based material system
 * Web application compatible by compiling to WebAssembly
 
-## Documents
+## Documentation
```

Comment:
English question. How Documents and Documentation are different?

Reply:
Documents are a multiple of document.
Documentation is a set of materials describing how to use, and the workings of a system, library or an API.

Documentation is a technical term, while documents are an office term.

In other words - all documentation are document(s), but not all documents are documentation.

Reply:
Thanks!